shell/platform/windows/flutter_windows_view.cc

```diff
@@ -163,6 +163,10 @@ void FlutterWindowsView::OnWindowSizeChanged(size_t width, size_t height) {
   }
 }
 
+void FlutterWindowsView::OnWindowRepaint() {
+  ForceRedraw();
```

Comment:
Please add a unit test in `shell/platform/windows/flutter_windows_view_unittests.cc` to make sure `OnWindowRepaint` sends a metric with the expected values. You should be able to reuse this one: https://github.com/flutter/engine/blob/695ac6832d9fba69b1f242f4a9a6ed4de4bcee61/shell/platform/windows/flutter_windows_view_unittests.cc#L550-L578

Reply:
done